scripts/sync_template_tracker.py

```diff
@@ -1,122 +1,212 @@
-#!/usr/bin/env python
-"""Populate template tracker with information based on cfme_data"""
+#!/usr/bin/env python3
+"""Populate template tracker with information based on cfme_data
+uses Pool.starmap, which is py3 only
+"""
 import sys
 from collections import defaultdict
-from threading import Lock
-from threading import Thread
+from collections import namedtuple
+from multiprocessing import Manager
+from multiprocessing.pool import ThreadPool
 
 from miq_version import TemplateName
-from slumber.exceptions import SlumberHttpBaseException
+from tabulate import tabulate
+from wrapanapi import EC2System
 from wrapanapi import Openshift
 
-from cfme.utils import net
 from cfme.utils import trackerbot
 from cfme.utils.conf import cfme_data
 from cfme.utils.log import add_stdout_handler
 from cfme.utils.log import logger
+from cfme.utils.path import log_path
 from cfme.utils.providers import get_mgmt
-from cfme.utils.providers import list_provider_keys
 
+# manager for thread queues, instead of locking
+manager = Manager()
 
-add_stdout_handler(logger)
+ProvTemplate = namedtuple('ProvTemplate', 'provider_key, template_name')
```

Comment:
typo

Reply:
I'm not seeing it, what's wrong?

Reply:
sorry, my bad

Reply:
Discussed offline - for clarity, Ievgen thought the fields for namedtuple had to be in iterable or passed as individual strings. namedtuple is loose with field names, and allows them to be passed like this.